### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by UdokaVrede**
This pull request makes the following changes:
-

Test checklist:
- [ ]

- [x] I certify that I ran my checklist

Fixes ushahidi/platform# .

Ping @ushahidi/platform

### CODE REVIEW

**Comment by UdokaVrede** [comment]
> [clabot:check]

[clabot:check]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 
-- **Documentation**: if you find an area of the Ushahidi platform that could use better docs, we would love to hear from you in an issue, and would be seriously excited if you send a [Pull Request](https://github.com/ushahidi/platform/compare). This is a great way to get involved even if you are not technical or just have a passion to make information more available and clear to everyone.
+### You can contribute to Ushahidi projects in these areas:
+
+- **Documentation**: if you find an area of the Ushahidi platform that could use better docs, we would love to hear from you in an issue, and would be seriously excited if you send a [Pull Request](https://github.com/ushahidi/platform/compare). This is a great way to get involved non-technically or if you are passionate about making information more available and clear to everyone.
```
Comment (rowasc) [inline CONTRIBUTING.md:6]: hey so this is totally not your fault, but could we change "non-technically" to "without knowing how to code" ? or just "...get involved if you are passionate...."  without mention of technical-ity :D 
Writing is a technical skill, and docs are tough, and I don't want folks to feel like their contribution is less technical or less valued <3

File: docs/contributors-to-ushahidi.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
 There are many ways to get involved with Ushahidi. You can read about how to contribute in [our docs](https://docs.ushahidi.com/platform-developer-documentation/contributing-or-getting-involved).
 
-This is a list of people who have contributed to Ushahidi. Not everyone in this list contributed code. Some folks here are designers, and they designed different components of the Ushahidi platform to make it better. Some did things that improved the organization and enabled others to thrive, like writing docs, blog posts, or perform business functions for our non profit. Many developed code and reviewed other people's code. A lot of people opened new bug tickets or provided ideas.
+This is a list of people who have contributed to Ushahidi. Not everyone in this list contributed in code. Some folks here are designers, and they designed different components of the Ushahidi platform to make it better. Some did things that improved the organization and enabled others to thrive, like writing docs, blog posts, or perform business functions for our non profit. Many developed codes and reviewed other people's code. A lot of people opened new bug tickets or provided ideas.
```
Comment (rowasc) [inline docs/contributors-to-ushahidi.md:5]: keep code singular (it's a difference in writing style only but it's the one we tend to use)

**Review by rowasc — CHANGES_REQUESTED** [review]
This is fantastic, thank you <3 I left a couple of very small requests, then we can merge

**Review by rowasc — APPROVED** [review]
LGTM! Thanks for the quick turnaround on applying feedback !

**Comment by rowasc** [comment]
@allcontributors please add @UdokaVrede for docs

**Comment by UdokaVrede** [comment]
Hi @rowasc can I go ahead to reading and improving the docs

**Pull request merged**